person.rb

```diff
@@ -1,4 +1,6 @@
-class Person
```

Comment:
- [ ] I have realized that there is something missing. A person has many rentals relationship seems not to have been handled. KIndly include it.

![image](https://user-images.githubusercontent.com/86318284/190379356-37a32aae-4b5d-44a2-b918-15977b5d8c30.png)